tools/data-handler/package.json

```diff
@@ -39,6 +39,7 @@
     "typescript-eslint": "^7.10.0"
   },
   "dependencies": {
+    "@mui/joy": "^5.0.0-beta.36",
```

Comment:
Why this is in d-h's package.json?

Shouldn't this be in app's?

Reply:
Yes it should